packages/frontend-v2/components/Plan/Plan.tsx

```diff
@@ -40,6 +43,8 @@ export const Plan: React.FC<PlanProps> = ({
     }
   };
 
+  const { setNodeRef } = useDroppable({ id: 'plan' });
```

Comment:
Here we make the plan a droppable. This is because our implementation of delete is performed when a schedule course is dragged over anything that isn't a droppable. Thus, we want to make it so that anytime a course is dragged over the plan, but not a term within the plan, it does not get deleted.